src/pages/computer-science/index.md

```diff
@@ -5,8 +5,8 @@ title: Computer Science
 
 Computer Science is the study of computers and the concepts that make computers possible.
 
-Much of computer science was pioneered in the latter half of the 20th century.
+Much of computer science was pioneered in the latter half of the 20th century. It has developed into one of the most pivotal fields, on which pretty much everything is dependent upon.
```

Comment:
I think that you should probably elaborate on this, since many people reading up on this field might not know how it affects the world.